python/cudf/cudf/tests/test_offset.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
```

Comment:
```suggestion
# Copyright (c) 2021, NVIDIA CORPORATION.
```

I think we will need to update copyright in the rest of the edited files too?

Reply:
Will fix